nix/haskell.nix

```diff
@@ -84,8 +84,7 @@ let
             prettyprinter-configurable.components.tests.prettyprinter-configurable-doctest.buildable = lib.mkForce false;
 
             language-plutus-core.components.benchmarks.language-plutus-core-create-cost-model = {
-              # Need a suitably wrapped R
-              build-tools = lib.mkForce [(pkgs.rWrapper.override { packages = with pkgs.rPackages; [tidyverse dplyr stringr MASS]; } )];
+              build-tools = with pkgs.rPackages; [pkgs.R tidyverse dplyr stringr MASS];
```

Comment:
:thinking: why is this different now? and different to the nix shell?

Reply:
Because that way, it'll show up properly when using R from within Haskell. This way the env variable for lib paths in R is set, instead of a wrapper binary which sets the lib paths (no wrapper binary invoked by `inline-r` from Haskell).

Reply:
Worth a comment.